### PR TITLE
Project page Styling

### PR DESCRIPTION
**Pull request by abehara2**
Deleted taking styling from multiple css sheets and refactored into page stylesheets

### CODE REVIEW

File: client/pages/editProject.jsx [hunk 1]
```diff
@@ -10,41 +10,40 @@ import {
...
-  //   return project.data[0].description;
-  // };
-  // getDescription().then(description => {
-  //   description = setDescription(description);
-  // });
+  //Get Title
+  const [title, setTitle] = useState("Loading");
+  const getTitle = async () => {
+    const project = await getModelsByID(id);
+    return project.data[0].name;
```
Comment (arpanlaha) [inline client/pages/editProject.jsx]: should probably handle the case where the backend request fails

File: client/pages/editProject.jsx [hunk 2]
```diff
@@ -10,41 +10,40 @@ import {
...
+  };
+  getTitle().then(name => {
```
Comment (arpanlaha) [inline client/pages/editProject.jsx]: we can do this async await style

File: client/pages/editProject.jsx [hunk 3]
```diff
@@ -10,41 +10,40 @@ import {
...
+    name = setTitle(name);
+  });
+  //Set Description
+  const [description, setDescription] = useState("Loading");
+  const getDescription = async () => {
+    const project = await getModelsByID(id);
+    return project.data[0].description;
+  };
+  getDescription().then(description => {
```
Comment (arpanlaha) [inline client/pages/editProject.jsx]: same as above

File: client/pages/editProject.jsx [hunk 4]
```diff
@@ -10,41 +10,40 @@ import {
...
+    return project.data[0].name;
+  };
+  getTitle().then(name => {
+    name = setTitle(name);
+  });
+  //Set Description
+  const [description, setDescription] = useState("Loading");
+  const getDescription = async () => {
+    const project = await getModelsByID(id);
+    return project.data[0].description;
```
Comment (arpanlaha) [inline client/pages/editProject.jsx]: would also break on backend request failure

File: client/pages/editProject.jsx [hunk 5]
```diff
@@ -10,41 +10,40 @@ import {
...
-  // //Set Description
-  // const getDescription = async () => {
-  //   const project = await getModelsByID("5e5f20e2f182430b94c8c66d");
-  //   return project.data[0].description;
-  // };
-  // getDescription().then(description => {
-  //   description = setDescription(description);
-  // });
+  //Get Title
+  const [title, setTitle] = useState("Loading");
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: Remove placeholder text

File: client/pages/editProject.jsx [hunk 6]
```diff
@@ -10,41 +10,40 @@ import {
...
+  const getTitle = async () => {
+    const project = await getModelsByID(id);
+    return project.data[0].name;
+  };
+  getTitle().then(name => {
+    name = setTitle(name);
+  });
+  //Set Description
+  const [description, setDescription] = useState("Loading");
```
Comment (yousefa00) [inline client/pages/editProject.jsx]: same here

**Pull request merged**